src/test/lrsql/admin/route_test.clj

```diff
@@ -113,11 +115,12 @@
                             (keys h/sec-head-names)))
 
 (deftest admin-routes-test
-  (let [sys  (support/test-system)
-        sys' (component/start sys)
+  (let [sys       (support/test-system)
+        sys'      (component/start sys)
```

Comment:
Extra space?

Reply:
nope, this is straight out of `clojure-align`. Look below

Reply:
OK my bad, the lrs config destructuring got in the way.